<launch>
    <arg name="platform" default="1.0" />

    <node pkg="iarc7_vision" type="iarc7_vision_node" name="iarc7_vision_node">

        <rosparam command="load" file="$(find iarc7_vision)/param/vision_node_params_$(arg platform).yaml" />

        <remap from="grid_line_estimator/pose" to="/camera_localized_pose" />
        <remap from="bottom_image_raw/image_raw" to="/bottom_camera/camera/image_raw" />
    </node>
</launch>
